Source/ACE.Server/Factories/LootGenerationFactory_Test.cs

```diff
@@ -193,7 +215,21 @@ public static LootStats LootStats(WorldObject wo, LootStats ls, bool logstats)
                         }
                         break;
                     case ItemType.Clothing:
-                        ls.ClothingCount++;
+                        if (testItem.Name.Contains("Cloak"))
```

Comment:
A minor concern would be picking up any other items with "Cloak" in the name, that aren't actually cloak

There is an IsCloak() function which could determine this more reliably